ocfweb/docs/docs/services/account/content-removal.md

```diff
@@ -0,0 +1,46 @@
+[[!meta title="Content Removal Policy"]]
```

Comment:
I feel "Removal Policy" or "Removal Requests" would look strange under the "Accounts" banner, it would seem like it refers to "Account Removal" instead of what it's actually about